### PR TITLE
refactor(cli): rewrite `info` in Rust

### PR DESCRIPTION
**Pull request by lucasfernog**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [ ] No

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Doesn't have colors for now, but we can add it later :)

### CODE REVIEW

**Review by nothingismagick — APPROVED** [review]

File: cli/tauri.js/package.json [hunk 1]
```diff
@@ -104,7 +102,7 @@
     "ts-loader": "8.0.18",
     "typescript": "4.2.3",
     "webpack": "5.26.3",
-    "webpack-cli": "4.5.0",
+    "webpack-cli": "^4.5.0",
```
Comment (nklayman) [inline cli/tauri.js/package.json]: Was this intentional?

**Review by nklayman — APPROVED** [review]
Works for me, nice job!

File: cli/tauri.js/package.json (hunk 1 above)
Comment (lucasfernog) [reply cli/tauri.js/package.json]: oops, renovate will fix it lol

Comment (lucasfernog) [reply cli/tauri.js/package.json]: oh I didn't merge it yet, I'll fix it lol

**Pull request merged**